config.yml

```diff
@@ -31,8 +31,11 @@ image-cache-url: ''
 terraform-scripts-location: 'terraform_test_artifacts'
 # Rancher provider version, leave empty for the latest version. e.g. '' or '3.1.1'.
 terraform-provider-rancher: ''
-# Harvester provider version, leave empty for the latest version. e.g. '' or '0.6.3'.
-terraform-provider-harvester: ''
+# Harvester provider version:
+# '' for the latest released version.
+# '0.0.0-dev' for the latest version from master branch.
```

Comment:
please notice that the *master branch* will require docker to build the provider while executing test case.

Reply:
Updated it. Thank you.